docs/test/test.md

```diff
@@ -10,28 +10,52 @@ This doc will give an introduction about the following three parts:
 
 # Organization of test
 
-Test in pouch could be divided into two parts [unit testing](https://en.wikipedia.org/wiki/Unit_testing#Description) and [integration testing](https://en.wikipedia.org/wiki/Integration_testing). More info could be get from the wiki page. For pouch developer, if your code is only used in a single module, then the unit test is enough. While if your code is called by multiple modules, integration tests are required. In pouch, both of them are developed using go language. More details could be got in [Unit Testing](#unit-testing) and [Integration Testing](#integration-testing).
+Test in pouch could be divided into following parts:
+
+* [unit testing](https://en.wikipedia.org/wiki/Unit_testing#Description)
+* [integration testing](https://en.wikipedia.org/wiki/Integration_testing)
+
+More info could be get from the wiki page. 
+
+For pouch developer, if your code is only used in a single module, then the unit test is enough. While if your code is called by multiple modules, integration tests are required. In pouch, both of them are developed using go language. More details could be got in [Unit Testing](#unit-testing) and [Integration Testing](#integration-testing).
 
 # Usage of Test
 
+Tests could be run through the target provided by `Makefile` in the root directory of pouch code. Also could be run manually.	
+To run the whole test, you could just run:
+
+```
+# make test
+```
+Please note that, in order to run test, the following prerequisites are needed:
+	
+	* golang is installed
+	* docker is installed
```

Comment:
In the future, we will replace docker with pouch itself.

Reply:
Yes, when pouch support build option~